packages/react-icons/build/pfIcons.js

```diff
@@ -0,0 +1,35 @@
+const {
+  save,
+  'folder-open': folderOpen,
+  edit,
+  print,
+  spinner,
+  home,
+  history,
+  memory,
+  server,
+  user,
+  users,
+  info,
+  filter,
+  key,
+  ...icons
+} = require('@patternfly/patternfly-next/icons/pf-icons');
+
+module.exports = {
+  pfIcons: {
+    'alt-save': save,
```

Comment:
these might be better as `save-alt` it un-groups them, but it does keep them next to the related icon when it comes to intellisense and the examples page.

Reply:
Yeah, good point!